tests/codeception/acceptance/StanfordCTAListCest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Codeception tests on CTA List paragraph type.
+ */
+class StanfordCTAListCest {
+
+  /**
+   *
+   */
+  protected function setupContentType(\AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/structure/types/manage/page/fields/add-field');
+    $I->selectOption('#edit-new-storage-type', 'Paragraph');
+    $I->selectOption('#edit-new-storage-type', 'Paragraph');
+    $I->seeElement('#edit-label');
+    $I->fillField('#edit-label', 'Paragraphs');
+    $I->click('//input[@class="button button--primary js-form-submit form-submit"]');
+    $I->seeElement('#edit-field-name');
+    $I->fillField('#edit-field-name', 'paragraphs');
+    $I->click('//input[@class="button button--primary js-form-submit form-submit"]');
+    $I->seeInCurrentUrl('node.page.field_paragraphs');
+    $I->click('//input[@id="edit-submit"]');
+    $I->seeElement('#edit-settings-handler-settings-negate-1');
+    $I->click('#edit-settings-handler-settings-negate-1');
+    $I->click('//input[@id="edit-submit"]');
+    drupal_flush_all_caches();
```

Comment:
As mentioned in slack. use the visible link text in 99% of scenarios. clicks and selects will search for the field labels and contents. The tests should be roughly human readable.

Reply:
I fixed this up.